rand_jitter/src/error.rs

```diff
@@ -54,13 +52,3 @@ impl ::std::error::Error for TimerError {
         self.description()
     }
 }
-
-impl From<TimerError> for Error {
-    fn from(err: TimerError) -> Error {
```

Comment:
I think we want this (and it's also not incompatible with the current Error type).